src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/Marshal.cs

```diff
@@ -582,6 +584,8 @@ public static void PtrToStructure<T>(IntPtr ptr, [DisallowNull] T structure)
             PtrToStructure(ptr, (object)structure!);
```

Comment:
Do we need `UnconditionalSuppressMessage` on `void PtrToStructure<T>(IntPtr ptr, [DisallowNull] T structure)` too?